bin/fetch.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env babel-node
 
+import "cross-fetch/polyfill"
```

Comment:
Where is it used? if shouldn't be imported there if it's not used in this file but used in one of the dependency